test/regen_code.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-bundle exec ruby src/rb/jack.rb test/test_project/project.yml src/java
+bundle exec ruby src/rb/jack.rb test/test_project/project.yml test/java
```

Comment:
We are moving the test project to `test` directory? If that's the case, we should remove the old test project code in the `src`. Otherwise, they will be duplicated.